src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -2,10 +2,13 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:context="http://www.springframework.org/schema/context"
        xsi:schemaLocation="http://www.springframework.org/schema/beans
-  http://www.springframework.org/schema/beans/spring-beans.xsd
-  http://www.springframework.org/schema/context
-  http://www.springframework.org/schema/context/spring-context.xsd"
->
-    <bean id="greetingService" class="org.noteskeeper.server.GreetingService"/>
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
```

Comment:
XML is widely used in the projects that started before Java Config appeared. As for new projects, I have faced only java configs. Main point is that, Spring boot doesn't support XML, so you would need to mix both approaches, that is not good.  As for just learning matters, it will be good to have some XML configs experience.
BTW, I do not see real benefits in XML Configs that may outweigh Java Config pros.